composer.json

```
@@ -16,9 +16,9 @@
         "psr-4": {"OckCyp\\CoversValidator\\Tests\\": "tests/"}
     },
     "require": {
-        "php": "^7.0 || ^8.0",
-        "phpunit/phpunit": "^6.0 || ^7.0 || ^8.0 || ^9.0",
-        "symfony/console": "^2.7 || ^3.0 || ^4.0 || ^5.0 || ^6.0"
+        "php": ">=7.1",
```

note that you're also making it compatible with PHP9 this way

---

some libraries seem to do it this way. I'll revert
```suggestion
        "php": "^7.0 || ^8.0",
```